torch/csrc/jit/runtime/static/passes.cpp

```diff
@@ -153,8 +154,78 @@ void FuseInferenceOpsForSparseNN(std::shared_ptr<torch::jit::Graph>& graph) {
   ClipRangesGatherRangesLengthsToOffsets(graph);
   ClipRangesGatherSigridHash(graph);
   ClipRangesGatherRangesSigridHash(graph);
+  ReplaceWithCopy(graph);
 #endif
 }
 
+static auto registry =
```

Comment:
You probably want to update this to the new registration API. It should look something like this:

```
TORCH_LIBRARY(static_runtime, m) {
  m.def("pure_inputs", []() -> at::Tensor { return at::randn({1}); });
  m.def("permute_copy",
    [](at::Tensor self, ArrayRef<int64_t> dims) -> at::Tensor {
                       at::Tensor out = at::empty_like(self);
                       at::native::copy_(out, self);
                       return out.permute(dims);
    });
 }
```

If you have other ops in the `static_runtime` namespace that are registered to the dispatcher, you might hit some problems. The "real" fix would be to migrate all of those ops to use the new registration API and put their definitions all inside of this `TORCH_LIBRARY` block. If that's a big ask though (if the ops are scattered across a lot of bunch of files, etc), then we have a `TORCH_LIBRARY_FRAGMENT` macro that you can use instead (at the cost of some code cleanliness 😄 )